File: app/src/generic/java/org/tasks/location/GeofenceApi.java
package org.tasks.location;

import javax.inject.Inject;
import org.jetbrains.annotations.Nullable;
import org.tasks.data.Place;

@SuppressWarnings("EmptyMethod")
public class GeofenceApi {

  @Inject
  public GeofenceApi() {}

  public void registerAll() {}

  public void update(@Nullable Place place) {}

  public void update(String place) {}

  public void update(long taskId) {}
}

File: app/src/generic/java/org/tasks/location/GeofenceApi.kt
package org.tasks.location

import org.tasks.data.Place
import javax.inject.Inject

@Suppress("UNUSED_PARAMETER")
class GeofenceApi @Inject constructor() {
    fun registerAll() {}
    fun update(place: Place?) {}
    fun update(place: String?) {}
    fun update(taskId: Long) {}
}